op-challenger/README.md

```diff
@@ -56,19 +46,21 @@ DISPUTE_GAME_FACTORY=$(jq -r .DisputeGameFactoryProxy .devnet/addresses.json)
   --num-confirmations 1
 ```
 
-The mnemonic and hd-path above is a prefunded address on the devnet. The challenger respond to any created games by
-posting the correct trace as the counter-claim. The scripts below can then be used to create and interact with games.
+The mnemonic and hd-path above is a prefunded address on the devnet.
+The challenger respond to any created games by posting the correct
+trace as the counter-claim. The scripts below can then be used to
+create and interact with games.
 
-## Scripts
+## Subcommands
 
-The [scripts](scripts) directory contains a collection of scripts to assist with manually creating and playing games.
-This are not intended to be used in production, only to support manual testing and to aid with understanding how
-dispute games work. They also serve as examples of how to use `cast` to manually interact with the dispute game
-contracts.
+The `op-challenger` has a few subcommands to interact with on-chain
+fault dispute games. The subcommands support game creation, performing
+game moves, and viewing fault dispute game data.They should not be
+used in production and are intended to provide convenient manual testing.
 
 ### Understanding Revert Reasons
 
-When actions performed by these scripts fails, they typically print a message that includes the
+When these actions fail, they typically print a message that includes the
```

Comment:
Is this still true when using the go bindings? I usually find the revert reason gets lost with the go bindings.

Reply:
Ah that's a good catch yes I believe that's true. Since the err handling is verbose in the subcommands now, we shouldn't need the user to interpret any errors.

Reply:
I've removed this section since it should be irrelevant now 😄